tests/ui/derive-arbitrary/non_arbitrary_param/expected

```diff
@@ -1,5 +1 @@
 error[E0277]: the trait bound `Void: kani::Arbitrary` is not satisfied
```

Comment:
I think part of the test is making sure the diagnostic message correctly points to the problematic line.

Can you try updating the `expected` file to:
```
error[E0277]: the trait bound `Void: kani::Arbitrary` is not satisfied
    |\
14  |     let _wrapper: Wrapper<Void> = kani::any();\
    |                                   ^^^^^^^^^ the trait `kani::Arbitrary` is not implemented for `Void`\
    |
```
?

Reply:
I added the message back.